metrics/alertmanager/tikv.rules.yml

```diff
@@ -14,7 +14,7 @@ groups:
       summary: TiKV memory used too fast
 
   - alert: TiKV_GC_can_not_work
-    expr: sum(increase(tikv_gcworker_gc_tasks_vec{task="gc"}[1d])) < 1
+    expr: sum(increase(tikv_storage_mvcc_gc_delete_versions_sum[1d])) < 1 and (sum(increase(tikv_gcworker_gc_tasks_vec{task="gc"}[1d])) < 1 or sum(increase(tikv_gc_compaction_filter_skip[1d])) < 1)
```

Comment:
Please add parentheses.

Reply:
I've confirmed with @hicqu that the expression is not correct. I think he will update it.